code/game/objects/structures/dropship_equipment.dm

```diff
@@ -59,11 +59,12 @@
 	installed_equipment = loaded_equipment
 	loaded_equipment.ship_base = src
 
+	/* TO-DO: REMOVE THIS STUPID WAY TO LINK EQUIPMENT TO SHUTTLES
 	for(var/obj/docking_port/mobile/marine_dropship/S in SSshuttle.dropships)
```

Comment:
hello todo
Wheres the replacement for this?

Reply:
I don't remember why I commented out the _code_, probably runtimes. Part of the project is to re-do how equipment is linked because it is incredibly stupid.